cmd/oci-runtime-tool/generate.go

```diff
@@ -461,6 +454,32 @@ func setupSpec(g *generate.Generator, context *cli.Context) error {
 		}
 	}
 
+	if context.IsSet("linux-namespace-add") {
+		namespaces := context.StringSlice("linux-namespace-add")
+		for _, ns := range namespaces {
+			name, path, err := parseNamespace(ns)
+			if err != nil {
+				return err
+			}
+			if err := g.AddOrReplaceLinuxNamespace(name, path); err != nil {
+				return err
+			}
+		}
+	}
+
+	if context.IsSet("linux-namespace-remove") {
+		namespaces := context.StringSlice("linux-namespace-remove")
+		for _, name := range namespaces {
+			if err := g.RemoveLinuxNamespace(name); err != nil {
+				return err
+			}
+		}
+	}
+
+	if context.IsSet("linux-namespace-remove-all") {
```

Comment:
This should be `context.Bool("linux-namespace-remove-all")`. If not, when linux-namespace-remove-all is false, ClearLinuxNamespaces function will still be called.